core/common/src/ChangedElements.ts

```diff
@@ -11,26 +11,57 @@ import { AxisAlignedBox3dProps } from "./geometry/Placement";
  * @module Entities
  */
 
-/** @internal */
+/** Type of change bitflags that show what changed on an element
+ * @public
+ */
 export enum TypeOfChange {
+  /** A property in the element changed */
   Property = 0b1,
+  /** The geometry stream of a geometric element changed */
   Geometry = 0b10,
+  /** The placement of a geometric element changed */
   Placement = 0b100,
+  /** Indirect change occurred to this element by a related instance */
   Indirect = 0b1000,
+  /** Hidden properties of the element changed */
   Hidden = 0b10000,
 }
 
-/** @internal */
+/** Changed elements found in a changeset or between a range of changesets
+ * @public
+ */
 export interface ChangedElements {
+  /** Ids of elements that changed */
   elements: Id64String[];
+  /** ECClassIds of elements that changed */
   classIds: Id64String[];
+  /** Operation codes, see [[DbOpcode]] */
   opcodes: number[];
+  /** Type of change bitflags, see [[TypeOfChange]] */
   type: number[];
+  /** Model Ids of the changed elements */
   modelIds?: Id64String[];
```

Comment:
Under what circumstances will this be undefined?

Reply:
If for some reason you are relying on an older agent that used to not export model Ids, same for all other nullable fields. Other fields (like properties, checksums and parent Ids) may also not be computed by an agent if it's configured to not do so for performance reasons.

Reply:
Please make the docs say that.

Reply:
Pushed